scripts/zones/Southern_San_dOria/npcs/Ominous_Cloud.lua

```diff
@@ -29,6 +29,22 @@ function onTrade(player,npc,trade)
         {2971, 5867}, -- inoshishinofuda
         {2972, 5868}, -- shikanofuda
         {2973, 5869}, -- chonofuda
+		
+		{17301, 6299}, -- shuriken
```

Comment:
Seems you’ve got tabs messing up the indent. Set your editor to use 4 spaces when you press the tab key or better yet, get editorconfig to automate it for you.

Reply:
@tristamgreen u should try Pycharm with lua addon, haven't had issues with indentations there 😸

Reply:
well, hopefully the newest commit fixed the tabulation errors.  i'd been without my PC for a week thanks to hurricane dorian and just got back to all this.